_harp/css/components/sections/_main-preview.scss

```diff
@@ -21,6 +22,7 @@
 #preview-player {
   margin: 0 auto 30px;
   width: 100%;
+  padding-top: $preview-video-ratio * 100%;
```

Comment:
Is there no way we can make this work with the built-in responsive features? It's kind of sad if we can't use those on our own site.